docs/manuals/core/scheduling/index.md

```diff
@@ -0,0 +1,278 @@
+# Taipy Scheduler
+
+!!! warning "Available in Taipy Enterprise edition"
+
+    This section is relevant only to the Enterprise edition of Taipy.
+
+This documentation focuses on providing necessary information to use the Taipy `Scheduler^`, and in particular
+the capabilities related to scenario management and job execution. It is assumed that the reader already knows the
+[Taipy Core entities](../entities/index.md) described in a previous chapter.
+
+Let’s assume also that the configuration represented in the picture below is implemented in the
+module <a href="./code_example/my_config.py" download>`my_config.py`</a>.
+
+=== "Module my_config.py"
+
+    ```python linenums="1"
+    {%
+    include-markdown "./code_example/my_config.py"
+    comments=false
+    %}
+    ```
+
+## Start/Stop the scheduler
+
+For the scheduler to run the scheduled methods, first you need to start the scheduler using the `start()` method.
+
+```python
+import taipy as tp
+
+tp.Scheduler.start()
+```
+
+Once the scheduler is started, the scheduled methods will run in the background at the scheduled time.
+While the scheduler is running, you can run other methods without being blocked by the scheduler.
```

Comment:
```suggestion
Once the scheduler is activated, the scheduled methods will run in the background at their specified times. While the scheduler is running, you can execute other methods without being held up by the scheduler.
```

Reply:
Same remark here.
I don't like the term `activated` to talk about a service.
It s still like a car or an engine. It is not activated, it is running.
What do you think?

Reply:
Yup, here I agree, that "activated" does not reflect directly that we want to say the scheduler is running. I have updated the commit, thanks.